web/cypress/integration/morecast-page.spec.ts

```diff
@@ -63,29 +80,57 @@ describe('MoreCast Page', () => {
     cy.url().should('contain', `${timeOfInterestQueryKey}=${timeOfInterest}`)
   })
 
+  describe('When loading a single station from url', () => {
+    beforeEach(() => {
+      interceptData()
+
+      cy.visit(`${MORECAST_ROUTE}?codes=${stationCode}`)
+      cy.wait('@getStations')
+    })
+
+    it('It should load with an observation table', () => {
+      cy.getByTestId(`observations-table-${stationCode}`)
+        .find('tbody > tr')
+        .should('have.length', 119)
```

Comment:
Should these lengths and widths be constants to avoid magic numbers?

Reply:
I've changed it to use a common constant for that variable.

I'm more bothered by the width test I do, but I don't have a nice answer for that.